### PR TITLE
Delete devices

### PR DESCRIPTION
**Pull request by kuby**
- Delete selected devices on the table
- Action buttons on the device (containing Details and Delete actions)
- Delete device from details view
- Added closest unit conversion for the timeToLive default value FYI @jkralik

### CODE REVIEW

**Review by jkralik — APPROVED** [review]

**Review by ondrejtomcik — APPROVED** [review]

File: http-gateway/web/src/containers/things/things-i18n.js [hunk 1]
```diff
@@ -152,7 +166,8 @@ export const messages = defineMessages({
   commandOnResourceExpired: {
     id: 'things.commandOnResourceExpired',
     defaultMessage: 'command on resource {deviceId}{href} has expired.',
-    description: 'Continuos message for command expiration, keep the first letter lowercase!',
+    description:
+      'Continuos message for command expiration, keep the first letter lowercase!',
```
Comment (oskarbauer) [inline http-gateway/web/src/containers/things/things-i18n.js:170]: Continuous message for command expiration, keep the first letter lowercase!

Comment (oskarbauer) [reply http-gateway/web/src/containers/things/things-i18n.js:170]: Not sure what this sentence means, and there is a typo.

File: http-gateway/web/src/containers/things/things-i18n.js [hunk 2]
```diff
@@ -310,4 +325,32 @@ export const messages = defineMessages({
...
     defaultMessage: 'Minimal value is {minimalValue}.',
   },
+  thingsDeleted: {
+    id: 'things.thingsDeleted',
+    defaultMessage: 'Things deleted',
+    description: 'Title of the toast message on things deleted success.',
+  },
+  thingsDeletedMessage: {
+    id: 'things.thingsDeletedMessage',
+    defaultMessage: 'The selected things were successfully deleted.',
```
Comment (oskarbauer) [inline http-gateway/web/src/containers/things/things-i18n.js]: should "things" be capitalized?

Comment (oskarbauer) [reply http-gateway/web/src/containers/things/things-i18n.js]: might be needed on all of the new translations.

File: http-gateway/web/src/containers/things/things-i18n.js [hunk 3]
```diff
@@ -310,4 +325,32 @@ export const messages = defineMessages({
...
+  },
+  thingDeleted: {
+    id: 'things.thingDeleted',
+    defaultMessage: 'Thing deleted',
+    description: 'Title of the toast message on thing deleted success.',
+  },
+  thingWasDeleted: {
+    id: 'things.thingWasDeleted',
+    defaultMessage: 'Things {name} was successfully deleted.',
```
Comment (oskarbauer) [inline http-gateway/web/src/containers/things/things-i18n.js]: Thing {name} was successfully deleted.

File: http-gateway/web/src/containers/things/things-list-page.js [hunk 4]
```diff
@@ -36,9 +98,43 @@ export const ThingsListPage = () => {
...
+            <i className="fas fa-trash-alt" />
+            {`${_(t.delete)} ${
+              selectedDevicesCount > 1
+                ? `${selectedDevicesCount} ${_(menuT.things)}`
+                : selectedDeviceName
+            }`}
+          </>
+        }
+        body={
+          <>
```
Comment (oskarbauer) [inline http-gateway/web/src/containers/things/things-list-page.js]: <> not needed

**Review by oskarbauer — APPROVED** [review]

File: http-gateway/web/src/containers/things/things-i18n.js (hunk 1 above)
Comment (kuby) [reply http-gateway/web/src/containers/things/things-i18n.js:170]: Its not a message, it's just a description of the translate message (for the translators).

**Pull request merged**